### PR TITLE
Synchronise config files with Laravel 6

### PR DESCRIPTION
**Pull request by bennothommo**
Replaces #171.

Laravel 6 has had updates to their config files since we released the 1.1 branch, this PR brings across these changes while still keeping the Winter CMS defaults and modifications intact. All new values in the configurations have been set to their defaults as per Laravel 6.

I have tested this with `artisan winter:env` and the environment variables are still populating correctly.

Thanks to @ericp-mrel for the initial PR.

### CODE REVIEW

File: config/cache.php [hunk 1]
```diff
@@ -11,11 +11,6 @@
     | using this caching library. This connection is used when another is
     | not explicitly specified when executing a given caching function.
     |
-    | WARNING! Do not use anything that is used for other information in your
-    | application. Example: If you are using redis for managing queues and / or
-    | sessions, you should NOT be using the EXACT SAME redis connection for the
-    | Cache store, as calling Cache::flush() will flush the entire redis store.
-    |
```
Comment (LukeTowers) [inline config/cache.php]: Revert this change please.

File: config/cache.php [hunk 2]
```diff
@@ -42,32 +40,49 @@
         ],
 
         'database' => [
-            'driver'     => 'database',
-            'table'      => 'cache',
+            'driver' => 'database',
+            'table' => 'cache',
```
Comment (LukeTowers) [inline config/cache.php]: Revert all the alignment changes please

File: config/database.php [hunk 3]
```diff
@@ -110,19 +120,24 @@
...
     | provides a richer set of commands than a typical key-value systems
-    | such as APC or Memcached. Laravel makes it easy to dig right in.
+    | such as APC or Memcached. Winter CMS makes it easy to dig right in.
     |
     */
 
     'redis' => [
 
         'client' => 'predis',
-        'cluster' => false,
```
Comment (LukeTowers) [inline config/database.php:120]: Is the code setup to still use this option if it was set somewhere?

File: config/queue.php [hunk 4]
```diff
@@ -4,15 +4,12 @@
 
     /*
     |--------------------------------------------------------------------------
-    | Default Queue Driver
+    | Default Queue Connection Name
     |--------------------------------------------------------------------------
     |
-    | The Laravel queue API supports a variety of back-ends via an unified
+    | Winter CMS's queue API supports an assortment of back-ends via a single
```
Comment (LukeTowers) [inline config/queue.php]:
```suggestion
    | The Winter CMS queue API supports an assortment of back-ends via a single
```

File: config/queue.php [hunk 5]
```diff
@@ -39,37 +38,32 @@
             'driver' => 'database',
             'table' => 'jobs',
             'queue' => 'default',
-            'expire' => 60,
```
Comment (LukeTowers) [inline config/queue.php:42]: Is this option still looked at in the code?

File: config/queue.php [hunk 6]
```diff
@@ -39,37 +38,32 @@
...
+            'retry_after' => 90,
         ],
 
         'beanstalkd' => [
             'driver' => 'beanstalkd',
             'host' => 'localhost',
             'queue' => 'default',
-            'ttr' => 60,
```
Comment (LukeTowers) [inline config/queue.php:49]: Is this option still looked at in the code?

**Review by LukeTowers — CHANGES_REQUESTED** [review]

File: config/database.php (hunk 3 above)
Comment (bennothommo) [reply config/database.php:120]: @LukeTowers nope, it was moved into the `options` array in Laravel, and we don't have any code that relies on this.

File: config/queue.php (hunk 5 above)
Comment (bennothommo) [reply config/queue.php:42]: @LukeTowers `expire` was renamed to `retry_after` - `expire` is not used anywhere.

File: config/queue.php (hunk 6 above)
Comment (bennothommo) [reply config/queue.php:49]: @LukeTowers `ttr` was renamed to `retry_after` like the database change above.

**Review by LukeTowers — APPROVED** [review]
